modules/shared/src/main/scala/io/constellationnetwork/schema/delegatedStake.scala

```diff
@@ -97,7 +97,7 @@ object delegatedStake {
   @derive(eqv, show, encoder)
   case class DelegatedStakeInfo(
     nodeId: PeerId,
-    acceptedOrdinal: SnapshotOrdinal,
+    acceptedOrdinal: Option[SnapshotOrdinal],
```

Comment:
So I think if we change the other state information, we can still preserve this information for determining the snapshot ID associated with the actual update. (It requires one change.) We need to either do that and revert this, or change the API docs to reflect the change.